types/convert_test.go

```diff
@@ -156,7 +156,9 @@ func (s *testTypeConvertSuite) TestConvertType(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(vv.(Duration).String(), Equals, "10:11:12.1")
 
-	vt, err := ParseTime(nil, "2010-10-10 10:11:11.12345", mysql.TypeTimestamp, 2)
+	vt, err := ParseTime(&stmtctx.StatementContext{
+		TimeZone: time.Local,
```

Comment:
ditto

Reply:
ParseTime doesn't need the `TimeZone`, only Timestamp need it, so I think `nil` is acceptable.

Reply:
Yes, ParseTime with `mysql.TypeDatetime` or `mysql.TypeDate` type `nil` is acceptable but required for `mysql.TypeTimestamp`